scripts/setup/dev_setup

```diff
@@ -3,4 +3,4 @@
 set -e -o pipefail
 
 # binary will be $(go env GOPATH)/bin/golangci-lint
-curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.45.2/install.sh | sh -s -- -b $(go env GOPATH)/bin
\ No newline at end of file
+curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/v1.64.8/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.64.8
```

Comment:
Does this change the devsetup behavior?

Reply:
the old version is not compatible with golang 1.24 anymore, the lint setup is the same as before